src/types/events.ts

```diff
@@ -81,7 +81,7 @@ export type OnTextTracksData = Readonly<{
 export type OnVideoTracksData = Readonly<{
   videoTracks: ReadonlyArray<
     Readonly<{
-      trackId: number;
+      trackId: string;
```

Comment:
Is this really necessary ?
It will be a breaking change for applications...

Reply:
I made trackId a string because the 'id' of the Format in Android is a string type. The exoplayer simply assigns the indices of this group as an ids for all format of the first video/audio group. And despite this they are also strings. Also if there is a second group, then the formats there will receive real ids from the playlist.

Actually, the trackId has no meaning, because not used anywhere.